ui/stratutils/src/main.ts

```diff
@@ -8,15 +8,15 @@ import { Rules } from 'stratops/types';
 // import { promotion } from 'stratutils'
 export * as promotion from './promotion';
 
-export const initialFen: Fen = 'rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1';
+export const initialFen: Fen = 'rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1'; // @TODO VFR fix THIS ???
```

Comment:
dont think this needs altering. if it does its something that effects all variants not just abalone